packages/components/src/components/date-picker/date-picker.css

```diff
@@ -230,7 +223,6 @@ duet-date-picker .duet-date__input:focus::placeholder {
     var(--telekom-spacing-unit-x1)
   );
   transition: all var(--telekom-motion-duration-transition)
-    /* FIXME:TOKEN update this */
     cubic-bezier(var(--telekom-motion-easing-standard));
```

Comment:
this is clear :) I want to keep `cubic-bezier(…)` for now to keep it working until we update the tokens package (since this is not strictly related to dark mode, we can leave it for last)